_data/server_permissions.yaml

```diff
@@ -0,0 +1,151 @@
+# general permissions
+- permission: adminMessageReceive
+  description: Receives messages sent to the admin channel
+- permission: adminMessageSend
+  description: Allows sending messages to the admin channel
+- permission: hideAdmin
+  description: Doesn't show `@`
+- permission: info
+  description:
```

Comment:
![image](https://user-images.githubusercontent.com/7908039/103838280-e6350080-5049-11eb-8ab9-19fbd676ca91.jpeg)

Reply:
Yes, however I don't know what it does. In the initial comment for the PR I mentioned it.

Reply:
I've removed `info`, I don't see it ever being used in our code base for any checks.

Reply:
Also removed `listPerms` which seems to be useless as well.